tests/e2e/specs/stories-editor/global-inserter.test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, getAllBlocks } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { activateExperience, deactivateExperience, insertBlock } from '../../utils';
+
+const INSERTER_SELECTOR = '#amp-story-inserter';
+const CTA_BLOCK_NAME = 'Call to Action';
+
+/**
+ * Tests that a given block is present in the inserter.
+ *
+ * @param {string} block The name of the block, like 'Image'.
+ */
+const testBlockPresentInInserter = async ( block ) => {
+	// It should be possible to insert the block.
+	await insertBlock( block );
```

Comment:
`insertBlock` should be already inserting the block using the global inserter, if that succeeds then this would already mean that the block is present in the inserter. Maybe it would make more sense to test if the block was actually added to the page after inserting? Alternatively, if you're testing if everything works without using `insertBlock` then maybe inserting the block after checking that there is a match would make more sense?

 Or perhaps I'm missing the intent of the test. Is there a case where `insertBlock` would succeed but the rest of the testing steps would get a different result?